src/server-core/src/lib.rs

```diff
@@ -287,6 +287,23 @@ pub struct ReloadingTlsConfig {
     pub mode: TlsMode,
 }
 
+pub type ReloadTrigger = BoxStream<'static, Option<oneshot::Sender<Result<(), anyhow::Error>>>>;
+
+/// Returns a ReloadTrigger that triggers once per hour.
+pub fn default_cert_reload_ticker() -> ReloadTrigger {
+    let ticker = IntervalStream::new(tokio::time::interval(Duration::from_secs(60 * 60)));
```

Comment:
Just a thought, would we ever want the rate at which we reload certs to be configurable via LaunchDarkly?

Reply:
Dunno! The cloud team seemed to think hourly was very sufficient. Hooking this up to LD might be tricky due to bootstrap ordering.

Reply:
Dunno! The cloud team seemed to think hourly was very sufficient. Hooking this up to LD might be tricky due to bootstrap ordering.